torch/testing/_internal/common_utils.py

```diff
@@ -691,12 +689,121 @@ def read_and_process():
             "Test is disabled because an issue exists disabling it: {}".format(disabled_test_from_issues[test_name]) +
             " To enable set the environment variable PYTORCH_RUN_DISABLED_TESTS=1")
 
+# Helper that returns the maximum value a dtype can represent
+def _get_width(dtype):
```

Comment:
unused?

Reply:
Unusued! Good catch. A stray from my attempts at alternative implementations of the type promotion logic.